### PR TITLE
BaSP-TG-70: Employees List

### PR DESCRIPTION
**Pull request by FiolAxel**
- Delete older component.
- Implement Table.
- Implement Buttons.
- Implement Styles.
- Implement Modal.

### CODE REVIEW

File: src/Components/Employees/index.js [hunk 1]
```diff
@@ -1,40 +1,96 @@
...
-      setList(data.data);
+      setData(data.data);
+      setUntouchedData(data.data);
+      setLoading(false);
     } catch (error) {
       console.error(error);
     }
   }, []);
 
   const deleteItem = async (_id) => {
```
Comment (laura-brussa12) [inline src/Components/Employees/index.js:30]: This function should not be async

File: src/Components/Employees/index.js [hunk 2]
```diff
@@ -1,40 +1,96 @@
...
+      setIsOpen(false);
     } catch (error) {
       console.error(error);
     }
-    setList(list.filter((listItem) => listItem._id !== _id));
+    setData(data.filter((employee) => employee._id !== idToDelete));
+    setUntouchedData(untouchedData.filter((employee) => employee._id !== idToDelete));
+  };
+
+  const column = [
```
Comment (laura-brussa12) [inline src/Components/Employees/index.js:54]: Maybe you should move this to line 16 to have all const together

File: src/Components/Employees/employees.module.css [hunk 3]
```diff
@@ -22,4 +22,23 @@
...
+.loading {
+ padding: 50%;
+ margin-left: 20%;
+}
+
+.addEmployee {
+  margin-right: auto;
+}
+
+.h2 {
```
Comment (HernanCazeneuve) [inline src/Components/Employees/employees.module.css:36]: try to use classnames  and not  element selectors.

File: src/Components/Employees/employees.module.css [hunk 4]
```diff
@@ -22,4 +22,23 @@
   background-color: #ffffff;
   transform: scale(1.05);
   transition: all 0.2s ease-in-out;
```
Comment (HernanCazeneuve) [inline src/Components/Employees/employees.module.css:24]: button hover is defined in the shared component

File: src/Components/Employees/employees.module.css [hunk 5]
```diff
@@ -1,9 +1,9 @@
 .container {
-  flex: 1;
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
+  margin-top: 7%;
 }
 
 .button {
```
Comment (HernanCazeneuve) [inline src/Components/Employees/employees.module.css:9]: also coud delete these style that is not used and defined in the shared component

**Review by HernanCazeneuve — APPROVED** [review]

**Review by Gearyandres94 — CHANGES_REQUESTED** [review]
This is how it looks the loading screen.
Asi se

**Comment by Gearyandres94** [comment]
The add new Employee button also doesnt work.

**Review by laura-brussa12 — APPROVED** [review]

**Review by Gearyandres94 — DISMISSED** [review]
Great Job!!!!

**Review by HernanCazeneuve — DISMISSED** [review]

**Review by Gearyandres94 — DISMISSED** [review]
Nice Job!!

**Review by HiginiaMed — DISMISSED** [review]

**Review by GuidoCerioni — APPROVED** [review]
Good job!

**Review by AlejandroSantini — DISMISSED** [review]
Good job!

**Review by HernanCazeneuve — APPROVED** [review]

**Review by AlejandroSantini — APPROVED** [review]
Exelent the styles!

**Review by HiginiaMed — APPROVED** [review]

**Review by HiginiaMed — APPROVED** [review]

**Pull request merged**